### PR TITLE
Bump tgtg to 0.14.0 and self to 5.0.0

### LINKED ISSUE #37: 403 error - Please enable JS and disable any ad blocker
Integration of tgtg is not working on my setup.

In the logs i see error 403, it responds with geo.captcha-delivery.com

Any ideas?

### PR DESCRIPTION
**Pull request by tjorim**
https://github.com/ahivert/tgtg-python/releases/tag/0.13.2
https://github.com/ahivert/tgtg-python/releases/tag/0.14.0

Fixes #37.

Also updated documentation and `tgtg_get_tokens` script.
Bumped major version because of cookie requirement.

### CODE REVIEW

**Comment by Chouffy** [comment]
Hi @tjorim, thanks a lot for your PR!
I see that you also added dependabot, however could you explain what does it do?
I have limited time unfortunately to deep dive into this :/

**Comment by tjorim** [comment]
No worries, dependabot is a GitHub tool that (in this case) checks for updates in GitHub actions and opens a PR for it automatically. I already bumped actions/checkout to v3 in this PR.

**Comment by skinkie** [comment]
I would say bump to 0.14.0 :-)

**Comment by tjorim** [comment]
@Chouffy Just tested out the latest changes and working nicely.
image
As users will need to request new credentials (to include the cookie) anyway,
I gave it a major version bump and changed the `username` option to `email` as well.

File: custom_components/tgtg/manifest.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "domain": "tgtg",
   "name": "TooGooToGo",
-  "version": "4.2.2",
+  "version": "5.0.0",
   "documentation": "https://github.com/Chouffy/home_assistant_tgtg",
   "issue_tracker": "https://github.com/Chouffy/home_assistant_tgtg/issues",
   "codeowners": ["@chouffy"],
-  "requirements": ["tgtg>=0.13.1"],
+  "requirements": ["tgtg==0.14.0"],
```
Comment (Chouffy) [inline custom_components/tgtg/manifest.json:8]: I wouldn't necessarily fix the version, but let's keep it for now

**Comment by Chouffy** [comment]
Thanks a lot @tjorim ! I'll right away do a release with the breaking change mentioned

**Pull request merged**